src/main/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperations.scala

```diff
@@ -49,13 +50,21 @@ object TezosDatabaseOperations extends LazyLogging {
     */
   def writeAccounts(
       accountsInfo: List[BlockTagged[Map[AccountId, Account]]]
-  )(implicit ec: ExecutionContext): DBIO[Int] = {
+  ): DBIO[Option[Int]] = {
+    import CustomPostgresProfile.api._
+
+    val keepMostRecent = (rows: List[Tables.AccountsRow]) =>
```

Comment:
If the purpose of the PR is how change *how* data is written but not *what* is written and in which order, how come we have to do this additional processing to figure out `keepMostRecent`?

Reply:
Because previously we sequentially updated each row, so keeping the right order would guarantee that overwrites were correct.
Now we're trying to optimise with a bulk upsert, so we don't want to have duplicates ids in the bulk query.